common/unit_test/Test_Utils.cpp

```diff
@@ -541,3 +541,24 @@ TEST(IndexSequence, 3Dto5D) {
   EXPECT_EQ(default_axes1, ref_axes1);
   EXPECT_EQ(default_axes2, ref_axes2);
 }
+
+TEST(ToArray, lvalue) {
+  std::array<int, 3> arr{1, 2, 3};
+  Kokkos::Array<int, 3> ref_arr{1, 2, 3};
+  auto kokkos_arr = KokkosFFT::Impl::to_array(arr);
+  for (std::size_t i = 0; i < arr.size(); ++i) {
+    ASSERT_EQ(kokkos_arr[i], ref_arr[i]);
+  }
+}
+
+TEST(ToArray, rvalue) {
+  Kokkos::Array<int, 3> ref_arr{1, 2, 3};
+  auto kokkos_arr = KokkosFFT::Impl::to_array(std::array{1, 2, 3});
+  for (std::size_t i = 0; i < ref_arr.size(); ++i) {
+    ASSERT_EQ(kokkos_arr[i], ref_arr[i]);
+  }
+  // [TO DO] Need to update Kokkos version for compile time test
+  // GTEST_SKIP() << "Skipping ToArray::rvalue compile time test";
+  // static_assert(KokkosFFT::Impl::to_array(std::array{1, 2}) ==
+  // Kokkos::Array{1, 2});
```

Comment:
If you want,  you can already implement the test using the comparison operator inside `ifdef` based on the version of Kokkos ?

Reply:
If you are fine, I would like to keep it in comments since we do not test that in CI for the moment.

Reply:
Yes I am ok with that. One question though, how do we keep track that we will need to update this snippet when we increase the minimum version of Kokkos ?

Reply:
What about tracking in an [issue](#85)?

Reply:
I am not really confident on remembering that there is actually an issue about this when we bump the minimum version. We can try.

Reply:
Let me update the internal Kokkos version. It would be the simplest way

Reply:
I have changed my mind to follow the Kokkos convention. https://github.com/kokkos/kokkos/blob/abcb3dce75effd5cd7a2d73e9e51a9202573acab/core/src/Kokkos_Array.hpp#L388-L396

What do you think?